src/routes.js

```diff
@@ -4,8 +4,8 @@ var routes = {},
     util = require('util');
 
 routes.root = function (req, res, db) {
-    db.Pod.allForList(function (pods) {
-        res.render('index.njk', { data: pods });
+    db.Pod.homeStats(function (home_stats) {
+        res.render('index.njk', { data: home_stats});
```

Comment:
Even if the code looks okay to me, the data are not displayed on the page with {{data.foo}}. Do I need to manually install nunchuck? Is this code working locally for you?